src/App.js

```diff
@@ -1,109 +1,62 @@
-import React, { useState, useEffect } from 'react'
-import Project from './components/Project'
-import TagCluster from './components/TagCluster'
-import { sortByTags, filterBySearch } from './utils'
-import Search from './components/Search'
-import Overlay from './components/Overlay'
-import styled from 'styled-components'
+import React from 'react'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import { useSpring, animated } from 'react-spring'
+import styled from 'styled-components/macro'
 
-export default function App() {
-  const [tags, setTags] = useState(null)
-  const [projects, setProjects] = useState(null)
-  const [searchTerm, setSearchTerm] = useState()
-  const [scrollPosition, setScrollPosition] = useState(0)
+import Opportunities from './components/Pages/Opportunities/Opportunities'
+import Navigation from './components/Navigation/Navigation'
+import CreateProject from './components/Pages/Create/CreateProject'
+import UserProjects from './components/Pages/UserProjects/UserProjects'
+import Dashboard from './components/Pages/Dashboard/Dashboard'
 
-  useEffect(() => {
-    fetch('https://unfinished-api.herokuapp.com/api/projects')
-      .then((res) => res.json())
-      .catch((error) => console.log(error))
-      .then((projects) => setProjects(projects))
-    fetch('https://unfinished-api.herokuapp.com/api/tags')
-      .then((res) => res.json())
-      .catch((error) => console.log(error))
-      .then((tags) => setTags(tags))
-  }, [])
-  useEffect(
-    () =>
-      setProjects(
-        (projects) =>
-          tags &&
-          projects &&
-          sortByTags(
-            tags.filter((tag) => tag.applies).map((tag) => tag.text),
-            projects
-          )
-      ),
-    [tags]
-  )
-  useEffect(
-    () => setProjects((projects) => searchTerm && projects && filterBySearch(searchTerm, projects)),
-    [searchTerm]
-  )
+import useNavigationSwipe from './hooks/useNavigationSwipe'
+
+export default function App() {
+  const [isNavigationVisible, setNavigationsVisible, swipe] = useNavigationSwipe()
+  const slide = useSpring({
+    top: isNavigationVisible ? '100px' : '0px',
```

Comment:
Nice, but is `0px` needed? Please try `0` instead.

Reply:
It is; when I tried 0 it just jumps between the start and end-state